usagov/vote-gov-nvrf-app
# VOTE-2954 Update to the latest PDF English

https://cm-jira.usa.gov/browse/VOTE-2954

Update to the latest English PDF from eac.gov

Testing:
1. Open the form locally
2. Go through the "update registration path" and fill out every field (both required and not required).
3. Confirm the PDF generates and is correctly filled out (download and new tab)
